_docs_integrate/data-model-overview.md

```diff
@@ -412,6 +432,18 @@ RequestItems can be sent inside of a Request and specify what should be done whe
+| --------------------- | ------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| @type                 | `"DeleteAttributeRequestItem"` | Specifies the type of the RequestItem for internal processing.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| attributeId           | `string`                       | The `id` of the own shared LocalAttribute you wish for the peer to delete.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
- "own shared Attribute" is more precise than "own shared LocalAttribute".
- You can also mention here that the `id` of the own shared Attribute matches the `id` of the corresponding peer shared Attribute.